context/app/static/js/components/Search/__tests__/accordionFilters.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+// eslint-disable-next-line import/no-unresolved
+import { render } from 'test-utils/functions';
+import { SearchkitProvider, SearchkitManager } from 'searchkit';
+
+import accordionFilters from '../accordionFilters';
+
+test('default export has the right keys', () => {
+  expect(Object.keys(accordionFilters).sort()).toEqual([
+    'AccordionCheckboxFilter',
+    'AccordionListFilter',
+    'AccordionRangeFilter',
+  ]);
+});
+
+test('empty AccordionListFilter', () => {
```

Comment:
Just testing that it renders?

Reply:
I had more here, but I think bringing in the special case to hide when there are no facets broke the originals. There should be more testing in, but I also wanted to get something out.